### PR TITLE
Add commands for context to the bug template

### PR DESCRIPTION
**Pull request by nrb**
Signed-off-by: Nolan Brubaker <nolan@heptio.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -5,12 +5,22 @@ about: Tell us about a problem you are experiencing
...
 
 
 **What did you expect to happen:**
 
 
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
+
+* `kubectl logs deployment/ark -n heptio-ark`
+* `ark backup describe <backupname>` or `kubectl get backup/<backupname> -n heptio-ark`
```
Comment (skriss) [inline .github/ISSUE_TEMPLATE/bug_report.md]: for kubectl command, add `-o yaml`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -5,12 +5,22 @@ about: Tell us about a problem you are experiencing
...
+* `ark backup logs <backupname>`
+* `ark restore describe <restorename>` or `kubectl get restore/<restorename> -n heptio-ark`
```
Comment (skriss) [inline .github/ISSUE_TEMPLATE/bug_report.md]: for kubectl command, add `-o yaml`

Comment (nrb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nrb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Comment by skriss** [comment]
LGTM.

**Pull request merged**